NetSDK/App.config

```diff
@@ -36,6 +36,9 @@
     <add key="TEST_IMPRESSIONS_URL_TEMPLATE" value="testImpressions/bulk" />
     <add key="METRICS_URL_TEMPLATE " value="metrics/{endpoint}" />
     <add key="SPLITS_STORAGE_CONCURRENCY_LEVEL" value="4" />
+    <add key="TEST_IMPRESSIONS_REFRESH_RATE" value=""/>
+    <add key="TEST_IMPRESSIONS_CACHE_SIZE_PER_TEST" value=""/>
```

Comment:
TEST_IMPRESSIONS_REFRESH_RATE is used by the impressions module to know the time interval where it should send bulk impressions to backend.
I'm also using TEST_IMPRESSIONS_CACHE_SIZE_PER_TEST, to send bulk impressions to backend when any of the splits has more registered impressions than expected/supported.